azkaban-common/src/test/resources/azkaban/executor/mail/faileExecutorHealthCheckMessage.html

```diff
@@ -0,0 +1,12 @@
+<h2 style="color:#FFA500"> Executor is unreachable. Executor host - executor1-host on Cluster - unit-tests</h2>
```

Comment:
The file name has typo. I think it is supposed to start with "failed" instead of "faile"

Reply:
Will update, thanks for catching this.

Reply:
Fixed.